tsdb/chunkenc/float_histogram.go

```diff
@@ -201,6 +202,10 @@ func (a *FloatHistogramAppender) GetCounterResetHeader() CounterResetHeader {
 	return CounterResetHeader(a.b.bytes()[2] & 0b11000000)
 }
 
+func (a *FloatHistogramAppender) setCounterResetHeader(cr CounterResetHeader) {
+	a.b.bytes()[2] = (a.b.bytes()[2] & 0b00111111) | (byte(cr) & 0b11000000)
+}
```

Comment:
Is this really different from the existing `setCounterResetHeader` helper function? If so, could you add a comment explaining the differences.

Reply:
I removed these and replaced it with calls through the appender, since we already have an appender at all invocations anyway.